packages/custom-field/test/visual/lumo/custom-field.test.js

```diff
@@ -239,4 +239,20 @@ describe('custom-field', () => {
       });
     });
   });
+
+  describe('whitespace theme', () => {
+    let layout;
+
+    beforeEach(() => {
+      layout = fixtureSync(`
+        <vaadin-custom-field theme="whitespace" label="Custom field">
+          <input type="text" />
+        </vaadin-custom-field>
+      `);
+    });
+
+    it('should correctly set label padding bottom', async () => {
+      await visualDiff(layout, 'custom-field-whitespace-theme');
+    });
+  });
```

Comment:
**nit:** How do you feel about turning this into a test case and then moving it into the `basic` section to simplify it a little bit?

```js
it('theme-whitespace', () => {
  layout.label = 'Custom field';
  layout.setAttribute('theme', 'whitespace');
  await visualDiff(layout, 'theme-whitespace');
});
```

References:

https://github.com/vaadin/web-components/blob/8da8f13f3d4c1c4c4a3094185a5a8ea61ec61017/packages/vertical-layout/test/visual/lumo/vertical-layout.test.js#L26-L29

https://github.com/vaadin/web-components/blob/8da8f13f3d4c1c4c4a3094185a5a8ea61ec61017/packages/avatar/test/visual/lumo/avatar.test.js#L40-L43

Reply:
That makes sense since we have similar things other places.

Reply:
Don't forget to remove the old screenshot: `packages/custom-field/test/visual/lumo/screenshots/custom-field/baseline/custom-field-whitespace-theme.png`.